src/test/java/com/microsoft/graph/httpcore/TelemetryHandlerTest.java

```diff
@@ -1,70 +0,0 @@
-package com.microsoft.graph.httpcore;
-
-import static org.junit.jupiter.api.Assertions.assertNotNull;
```

Comment:
we should have unit tests for the telemetry handler

Reply:
working on that now